<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiamonIssueRetBroDialog</class>
 <widget class="QDialog" name="DiamonIssueRetBroDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableWidget" name="historyTableWidget"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="7" column="0" colspan="3">
        <widget class="QPushButton" name="addPushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="jobNoLabel_3">
         <property name="text">
          <string>Quntity</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QComboBox" name="sizeComboBox"/>
       </item>
       <item row="8" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="stoneRadioButton">
         <property name="text">
          <string>Stone</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="diamondRadioButton">
         <property name="text">
          <string>Diamond</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="jobNoLabel_2">
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="quantityLineEdit"/>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="otherRadioButton">
         <property name="text">
          <string>Other</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QComboBox" name="typeComboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="jobNoLabel">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLineEdit" name="weightLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Weight</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
